pytorch/buildspec-sagemaker.yml

```diff
@@ -0,0 +1,101 @@
+account_id: &ACCOUNT_ID <set-$ACCOUNT_ID-in-environment>
```

Comment:
Let's rename this file to buildspec-sagemaker-lite-1-13.yml, and also build sagemaker-lite DLCs in the images specified below.

Reply:
Renamed.